fhir-bucket/src/main/java/com/ibm/fhir/bucket/app/Main.java

```diff
@@ -284,18 +328,83 @@ public void parseArgs(String[] args) {
                     throw new IllegalArgumentException("missing value for --path-prefix");
                 }
                 break;
+            case "--target-bucket":
+                if (i < args.length + 1) {
+                    this.targetBucket = args[++i];
+                } else {
+                    throw new IllegalArgumentException("missing value for --target-bucket");
+                }
+                break;
```

Comment:
Where is the readme for this? There is a lot of surface area, and I'm afraid you are the only one who knows what they all do.

Reply:
Should be in the base directory, but probably warrants being a guide. Will  need to add additional content to cover the ever-expanding list of capabilities

Reply:
Should there be a backlog issue for this?

Reply:
marking unresolved so we don't lose the need for an issue.

Reply:
Issue is #1633